apollo-federation/src/sources/connect/validation/extended_type.rs

```diff
@@ -214,6 +216,15 @@ fn validate_field(
         )
     });
 
+    if let Some((_, body)) = http_arg
+        .iter()
+        .find(|(name, _)| name == &CONNECT_BODY_ARGUMENT_NAME)
```

Comment:
Should this be using the `get_http_body_arg` function?

Reply:
removed, good catch!